dttools/src/category.c

```diff
@@ -133,6 +136,14 @@ void category_specify_first_allocation_guess(struct category *c, const struct rm
 	rmsummary_merge_max(c->first_allocation, s);
 }
 
+int category_in_bucketing_mode(struct category* c)
+{
+    if (c->allocation_mode == CATEGORY_ALLOCATION_MODE_GREEDY_BUCKETING ||
```

Comment:
BUCKETING_MODE_GREEDY, etc?

Reply:
BUCKETING_MODE_GREEDY is used internally to decide which algorithm to compute the next bucketing state [here](https://github.com/cooperative-computing-lab/cctools/blob/d2b0dc17a2b70de7091f35876d343bf0aa3c01a7/dttools/src/bucketing.c#L111). Category code I think should still use CATEGORY_ALLOCATION_...